lib/ModelMixins/CatalogMemberMixin.ts

```diff
@@ -95,7 +95,7 @@ function CatalogMemberMixin<T extends Constructor<CatalogMember>>(Base: T) {
 
     @computed
     get nameInCatalog(): string | undefined {
-      return super.nameInCatalog || this.name;
+      return super.nameInCatalog || this.name || this.uniqueId;
```

Comment:
`nameInCatalog` should only be used in the catalog. It's not the official name of an item. Maybe  `name` should use `uniqueId` if there isn't a name present, and then `nameInCatalog` can stay as it's original definition.